updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -0,0 +1,169 @@
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#oc-mirror-imageset-config-params_mirroring-ocp-image-repository[Image set configuration parameters]
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#oc-mirror-image-set-examples_mirroring-ocp-image-repository[Image set configuration examples]
+* xref:../../updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc#update-service-overview_updating-restricted-network-cluster-osus[About the OpenShift Update Service]
+
+[id="mirroring-image-set"]
+=== Mirroring an image set to a mirror registry
+
+You can use the oc-mirror CLI plugin to mirror images to a mirror registry in a xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-image-set-partial[partially disconnected environment] or in a xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-image-set-full[fully disconnected environment].
+
+These procedures assume that you already have your mirror registry set up.
+
+[id="mirroring-image-set-partial"]
+==== Mirroring an image set in a partially disconnected environment
+
+In a partially disconnected environment, you can mirror an image set directly to the target mirror registry.
+
+// Mirroring from mirror to mirror
+include::modules/oc-mirror-mirror-to-mirror.adoc[leveloffset=+4]
+
+[id="mirroring-image-set-full"]
+==== Mirroring an image set in a fully disconnected environment
+
+To mirror an image set in a fully disconnected environment, you must first xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#oc-mirror-mirror-to-disk_mirroring-ocp-image-repository[mirror the image set to disk], then xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#oc-mirror-disk-to-mirror_mirroring-ocp-image-repository[mirror the image set file on disk to a mirror].
+
+// Mirroring from mirror to disk
+include::modules/oc-mirror-mirror-to-disk.adoc[leveloffset=+4]
+
+// Mirroring from disk to mirror in a disconnected environment
+include::modules/oc-mirror-disk-to-mirror.adoc[leveloffset=+4]
+
+// Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster
+include::modules/oc-mirror-updating-cluster-manifests.adoc[leveloffset=+2]
+
+[id="updating-mirror-registry-content"]
+=== Keeping your mirror registry content updated
+
+After your target mirror registry is populated with the initial image set, be sure to update it regularly so that it has the latest content. You can optionally set up a cron job, if possible, so that the mirror registry is updated on a regular basis.
+
+Ensure that you update your image set configuration to add or remove {product-title} and Operator releases as necessary. Any images that are removed are pruned from the mirror registry.
```

Comment:
If it's okay with you, I'll implement this but keep the last two words as "mirror registry". 

I think that's the terminology that's been consistently used throughout the rest of this page, and I don't know enough to be sure that changing it would be more helpful to users. Let me know if you think I should still change it to "image registry".

Reply:
Feel free to use your best judgement :)

Reply:
If it isn't otherwise clear, the suggestions are suggestions. I always try to back up actual style guide/mod docs rules with a link to the rule set.